ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/FilterChips.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { Globe } from 'react-feather';
+import { Flex, FlexItem, ChipGroup, Chip } from '@patternfly/react-core';
+
+import { SearchFilter } from 'types/search';
+import { VulnerabilitySeverityLabel, FixableStatus, DefaultFilters } from './types';
+
+import './FilterChips.css';
+
+type FilterChipProps = {
+    isGlobal?: boolean;
+    name: string;
+};
+
+function FilterChip({ isGlobal, name }: FilterChipProps) {
+    if (isGlobal) {
+        return (
+            <Flex alignItems={{ default: 'alignItemsCenter' }} flexWrap={{ default: 'nowrap' }}>
+                <Globe height="15px" />
+                {name}
+            </Flex>
+        );
+    }
+    return <Flex>{name}</Flex>;
+}
+
+type FilterChipsProps = {
+    defaultFilters: DefaultFilters;
+    searchFilter: SearchFilter;
+    onDeleteGroup: (category) => void;
+    onDelete: (category, chip) => void;
+};
+
+function FilterChips({ defaultFilters, searchFilter, onDeleteGroup, onDelete }: FilterChipsProps) {
+    const deployments = (searchFilter.DEPLOYMENT as string[]) || [];
```

Comment:
Probably unlikely, but if a user enters a query param in the URL in the form of `s[Deployment]=some-value`, or if a bad link is created elsewhere in the app, this entire page will crash. We should make a note or ticket to try and remove the type assertions as a follow up. I've got a couple of functions in the `searchUtils` filter in an in-progress branch that I think would work for this.